ale_py/ale_python_interface.py

```diff
@@ -2,15 +2,46 @@
 # Author: Ben Goodrich
 # This directly implements a python version of the arcade learning
 # environment interface.
-__all__ = ['ALEInterface']
+__all__ = ["ALEInterface"]
+
+import os
+import sys
+import warnings
+import platform
 
-from ctypes import *
 import numpy as np
+
+from ctypes import *
 from numpy.ctypeslib import as_ctypes
-import os
 
-ale_lib = cdll.LoadLibrary(os.path.join(os.path.dirname(__file__),
-                                        'libale_c.so'))
+
+def _load_cdll(path, name):
+    if sys.platform.startswith("linux"):
+        ext_suffix = ".so"
+    elif sys.platform.startswith("darwin"):
+        ext_suffix = ".dylib"
+    elif sys.platform.startswith("win"):
+        ext_suffix = ".dll"
+    else:
+        raise RuntimeError(
+            'Platform "%s" not recognized while trying to resolve shared library'
+            % sys.platform
```

Comment:
This is a place where Google Python style seems to disagree with PEP-8, but: can you move the closing parentheses to be on the same line as the last part of their statement? The cognitive dissonance is too strong. See http://google.github.io/styleguide/pyguide.html

Reply:
I changed both of the exceptions to use `format` for consistency and fixed the parentheses.